src/skmultiflow/meta/leverage_bagging.py

```diff
@@ -9,52 +9,62 @@
+    return LeveragingBaggingClassifier(base_estimator=base_estimator,
+                                       n_estimators=n_estimators,
+                                       w=w,
+                                       delta=delta,
+                                       enable_code_matrix=enable_code_matrix,
+                                       leverage_algorithm=leverage_algorithm,
+                                       random_state=random_state)
+
+
+class LeveragingBaggingClassifier(BaseSKMObject, ClassifierMixin, MetaEstimatorMixin):
+    """ Leveraging Bagging ensemble classifier.
 
     Parameters
     ----------
-    base_estimator: skmultiflow.core.BaseSKMObject or sklearn.BaseEstimator (default=KNN)
+    base_estimator: skmultiflow.core.BaseSKMObject or sklearn.BaseEstimator \
+    (default=KNN)
         Each member of the ensemble is an instance of the base estimator.
         
     n_estimators: int (default=10)
         The size of the ensemble, in other words, how many classifiers to train.
 
     w: int (default=6)
-        The poisson distribution's parameter, which is used to simulate re-sampling.
+        The poisson distribution's parameter, which is used to simulate
+        re-sampling.
         
     delta: float (default=0.002)
         The delta parameter for the ADWIN change detector.
     
     enable_code_matrix: bool (default=False)
-        If set, it will enable the output detection code matrix.
+        If set, enables Leveraging Bagging MC using Random Output Codes.
     
     leverage_algorithm: string (default='leveraging_bag')
-        The bagging algorithm to use. Can be one of the following: 'leveraging_bag',
-         'leveraging_bag_me', 'leveraging_bag_half', 'leveraging_bag_wt', 'leveraging_subag'
+        | The bagging algorithm to use. Can be one of the following:
+        | 'leveraging_bag' - Leveraging Bagging using ADWIN
+        | 'leveraging_bag_me' - Assigns to a sample ``weight=1`` if \
+        misclassified, otherwise ``weight=error/(1-error)``
+        | 'leveraging_bag_half' - Use resampling without replacement for half \
+        of the instances
+        | 'leveraging_bag_wt' - Without taking out all instances
+        | 'leveraging_subag' - Using resampling without replacement
 
     random_state: int, RandomState instance or None, optional (default=None)
         If int, random_state is the seed used by the random number generator;
         If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used by `np.random`.
+        If None, the random number generator is the RandomState instance used
+        by `np.random`.
     
     Raises
     ------
-    ValueError: A ValueError is raised if the 'classes' parameter is
-    not passed in the first partial_fit call.
+    ValueError: A ValueError is raised if the 'classes' parameter is not \
+    passed in the first ``partial_fit call``.
```

Comment:
Should not the markdown end after `partial_fit`?

Reply:
Fixed in e979d59d7b7aee316c049a64d4540c07114848bc